internal/testhelpers/resources.go

```diff
@@ -49,6 +49,83 @@ func buildPodTemplateSpec(mainPodSleep int) corev1.PodTemplateSpec {
 	}
 }
 
+func BuildSecret(secretName, userKey, user, passwordKey, password, dbNameKey, dbName string) corev1.Secret {
```

Comment:
Exported functions should have a docstring, even if it's short and internal to the project.

Reply:
Fixed. I added a few more too.